autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+    '@typescript-eslint',
+    'import',
+  ],
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
```

Comment:
You have `extends` defined in two places, so I believe the second one is overwriting the first. Thus, the `airbnb` coding conventions are being applied

Reply:
Ah, that makes sense. I forgot that I had already defined `extends` when I wrote the second one.